libs/design/src/molecules/article/article/article.component.scss

```diff
@@ -0,0 +1,119 @@
+@import '../../../scss/daff-util';
+
+.daff-article {
+	display: block;
+
+	&__lead {
+		font-size: $medium-font-size;
+		font-weight: normal;
+
+		code {
+			font-size: $medium-font-size;
+		}
+	}
+
+	&__meta {
+		font-size: $small-font-size;
+	}
+
+	> h1,
```

Comment:
You can reduce this down by doing:

```
>  {
 some-tag {
  }
}
```

That way you don't have to repeat the direct descendant.

Reply:
done